### ISSUE TITLE
Request: reconstruct deleted chats using moderation chat history

### ISSUE THREAD

**Issue by LucilleTea** [lay295/TwitchDownloader#737]
### Checklist

- [X] I have checked the [issue page](https://github.com/lay295/TwitchDownloader/issues?q=is%3Aissue) for duplicates
- [X] I am running the latest version ([download here](https://github.com/lay295/TwitchDownloader/releases/latest))

### Write your feature request here

I have a lot of my old vods downloaded from before I knew about this software, and no chat to go with them. I understand when the vod is deleted from twitch the chat replay is too. However, twitch moderation tools allow you to view each chatter's entire chat history. Given a list of chatters to check, and a time frame the stream happened in, could it be possible to reconstruct a chat replay for an old VOD?

I don't know if this chat history API is public facing, and inputting the time frames manually might be annoying, but it would be lovely for preservation if at all possible.

**Comment by ScrubN** [comment]
Sounds like something a bit beyond the scope of TwitchDownloader, however I have been working on another project to implement these types chat parsing requests in a more verbose application. If user message history is exposed in the API then maybe it could be incorporated into the project. See #629 for more info about the project

**Comment by LucilleTea** [comment]
Yeah, it does sound like a better fit for a separate program really. Hopefully your project goes well!
